setup.py

```diff
@@ -55,7 +55,8 @@
                       'rlp', 'sha3', 'leveldb',
                       'ioflo==1.5.4', 'semver', 'base58', 'orderedset',
                       'sortedcontainers==1.5.7', 'psutil', 'pip',
-                      'portalocker==0.5.7', 'pyzmq', 'raet',
+                      'portalocker==0.5.7', 'pyzmq',
+                      'six', 'ioflo', 'libnacl',
```

Comment:
Duplicated `ioflo`; why do we need `six`? 
Did we extend our deb packages creation for new dependencies (nacl?)

Reply:
Fixed